rpcs3/Emu/NP/rpcn_client.h

```diff
@@ -110,7 +110,17 @@ class vec_stream
 		}
 
 		SceNpCommunicationId com_id{};
-		std::memcpy(&com_id.data[0], &vec[i], COMMUNICATION_ID_SIZE);
+		std::memcpy(&com_id.data[0], &vec[i], 9);
+		const std::string sub_id(reinterpret_cast<const char*>(&vec[i + 10]), 2);
```

Comment:
Maybe this is easier to understand
```suggestion
		std::memcpy(&com_id.data[0], &vec[i], sizeof(com_id.data) - 1);
		const std::string sub_id(reinterpret_cast<const char*>(&vec[i + sizeof(com_id.data)]), sizeof(com_id) - sizeof(com_id.data));
```